tests/python/unittest/test_runtime_profiling.py

```diff
@@ -257,6 +259,50 @@ def test_profile_function(target, dev):
     assert report[metric].value > 0
 
 
+@tvm.testing.parametrize_targets("llvm")
+def test_estimate_peak_fma_flops(target, dev):
+    # This test uses vectorized instructions so we need a target that supports them
+    if target == "llvm":
+        target = "llvm -mattr=+fma,+avx2"
```

Comment:
Is it possible to turn off vectorization and see if that affects flops as a test?

Reply:
This test also will require processors which support these extensions right? So I can't run this on my m1 mac. can we skip this if i do not support avx2?

Reply:
Turning off vectorization does have an effect. It drops flops down to ~1MFLOP/s.

Not sure how to check if the processor supports an instruction, I'll looked through the codebase and couldn't find any examples. avx2 and fma are at least 10 years old, so I think we can assume x86 targets support them (totally could be wrong here though).

Reply:
I agree for x86 it's probably safe, but in general as ARM gets more popular we should try to come up with a way to deal with this. I dunno, probs best ask others

Reply:
Yeah so this runs fine on m1 mac. I get around 2e11 FLOPS. 

However this is with just 'llvm' target reaching in which is interesting. Changing the target string to any of the following does not change results, at least not noticeably. It does output some things

"llvm"
"llvm -mcpu=apple-latest -mtriple=arm64-apple-macos"

Additionally 
"llvm -mattr=+fma,+avx2" generates the following printout
```
tests/python/unittest/test_runtime_profiling.py '+fma' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
```

Can't grep for the string above so Imma assume this is just llvm carrying TVM somehow

Reply:
`test_vm()` does seg fault on my machine but it's probably unrelated im guessing.

Reply:
I don't think `llvm -mcpu=apple-latest -mtriple=arm64-apple-macos` is the correct target for macOS. It is missing `-device=arm_cpu`.